README.md

```diff
@@ -1,2 +1,32 @@
 # lando-util
```

Comment:
Can you add a statement with the purpose and context of this repo and why it's structured this way? Specifically how/why it's done this way since lando already does these things. I surmise it has something to do with running tasks as containers in a k8s cluster, but don't know if this will be folded back into lando worker or not.